frontend/src/templates/GuideMainPageTemplate/GuideMainPageTemplate.jsx

```
@@ -33,17 +39,45 @@ const GuideMainPageTemplate = ({
+                  onRowsPerPageChange={setRowsPerPage}
+                  items={items}
+                />
+              )}
+            </ContentArea>
+            <div
+              className="tour-info-container"
+              style={{ maxWidth: 'fit-content' }}
+            >
+              <h4>{heading}</h4>
+              <p
+                dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)